src/main/scala/ai/privado/tagger/sink/DependencyNodeTagger.scala

```diff
@@ -0,0 +1,34 @@
+package ai.privado.tagger.sink
+
+import ai.privado.cache.RuleCache
+import ai.privado.inputprocessor.DependencyInfo
+import ai.privado.passes.Utility
+import ai.privado.tagger.PrivadoSimpleCpgPass
+import ai.privado.utility.Utilities
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.semanticcpg.language.*
+import org.slf4j.LoggerFactory
+
+class DependencyNodeTagger(cpg: Cpg, dependencies: List[DependencyInfo], ruleCache: RuleCache)
+    extends PrivadoSimpleCpgPass(cpg)
```

Comment:
Any reason to use Simple Pass instead of Parallel? If any, please add a comment in the pass

Reply:
done